### ISSUE TITLE
Add Noto JP font(s)

### ISSUE THREAD

**Issue by Andrew15-5** [typst/webapp-issues#477]
### Description

There is `Noto Serif CJK SC`, but there is no `Noto Serif CJK JP`. Perhaps the same is for `Sans` font family.

As a side note, since [there are a bazillion Noto fonts](https://discord.com/channels/1054443721975922748/1088371867913572452/1300475945538289714), at least showing them when autocompleting a font name in the editor would be nice.

### Browsers

Firefox

### OS

Linux

**Comment by laurmaedje** [comment]
Autocompletion does show the Noto fonts.

**Comment by Andrew15-5** [comment]
Indeed. It didn't show up the completion menu for me last time I tried. Now it does.

**Comment by Andrew15-5** [comment]
I think it didn't work because of this: https://github.com/typst/typst/issues/5436.

**Comment by laurmaedje** [comment]
<img width="395" alt="Image" src="https://github.com/user-attachments/assets/e9f05126-9e33-4748-8d94-d73adf865b7c" />
